src/syscheckd/syscheck.h

```diff
@@ -495,10 +495,17 @@ void audit_set_db_consistency(void);
 /**
  * @brief Check if Auditd is installed and running
  *
- * @return The PID of Auditd
+ * @return -1 on error, 0 when Auditd is disabled, 1 when Auditd is enabled and 2 when it is enabled and
+ * in immutable mode
```

Comment:
We should discuss using an enum here (instead of returning 0 return AUDIT_DIS). This will help us improve code readability.